### ISSUE TITLE
Requesting changes to be compatible with charmbracelet/bubbles >= v0.19.0

### ISSUE THREAD

**Issue by nikklassen** [Evertras/bubble-table#179]
There was a recent change to charmbracelet's bubble (charmbracelet/bubbles#584) that changed a function signature to be generic. Since you are using `key.Matches` this library is no longer compatible with future versions of `bubbles`. Can we bump the Go version of this module, or change the usage of `key.Matches`? It looks like the charmbracelet libraries are on Go 1.18 now, so it probably makes sense to target that here too.

**Comment by Evertras** [comment]
Hmmm good catch.  Currently it's being tested and working on go 1.13 and up, but honestly I'd be very surprised if anyone needs to be as low as 1.17 at this point with this library.  And with Bubble Tea itself bumping up, it's hard to argue that we need to support lower versions.

I'll take a look at sorting this out in the near future when I get some time.

**Comment by ashmckenzie** [comment]
Thanks @nikklassen for the detail on the cause of the problem 🙇 

I've created https://github.com/Evertras/bubble-table/pull/180 which passes tests locally and works when I use in my local app.

**Comment by ashmckenzie** [comment]
@nikklassen when you have a moment, would you mind testing https://github.com/ashmckenzie/bubble-table/tree/ashmckenzie/fix-bubbles-compatibility to see if it works for you? 

You should be able to run just the following to try my fork:

```sh
go mod edit -replace github.com/evertras/bubble-table=github.com/ashmckenzie/bubble-table@e729cd674896514f0a6a2712b6274f6e229c0e5
go mod tidy
```

**Comment by nikklassen** [comment]
I can try out the fix this weekend

**Comment by nikklassen** [comment]
@ashmckenzie yup, looks good

**Comment by ashmckenzie** [comment]
Thanks @nikklassen 👍

**Comment by Evertras** [comment]
Released as v0.17.0, thanks for the fix! Apologies for slowness, should be more around now.
